tests/src/com.google.protobuf/protobuf-java-util/3.21.12/src/test/resources/META-INF/native-image/protobuf-java-util-test-metadata/reflect-config.json

```diff
@@ -0,0 +1,6379 @@
+[
```

Comment:
- This is generated by conditional agent mode from the same `user-code-filter.json`. Since there are some `name` entries that has nothing to do with the dependent library, I stripped it to this file.